_commands/basics/history.md

```diff
@@ -0,0 +1,46 @@
+---
+---
+
+history
+--
+
+`history` is used to list command history. By default, `history` will print the last 500 commands typed into the terminal.
+~~~ bash
+$ history
+  ...
+  498  third_most_recent_command
+  499  second_most_recent_command
+  500  most_recent_command
+  501  history
+~~~
+
+<!--more-->
+
+### Useful Options
+
+#### `history | grep <search_string>`
+This option allows you to search for most recent commands containing the search string
+~~~ bash
+$ history | grep cd
+    9  cd /home/name/secrets
+   11  cd topSecrets
+   14  history | grep cd
+~~~
+#### `!11`
```

Comment:
Sorry if this was ambiguous – `!<#>` will re-run that command from your history. So in your case, `!11` would `cd topSecrets`.